Frontend/app/src/main/java/objects/Exercise.java

```diff
@@ -40,4 +40,9 @@ public String getSet() {
     public void setSet(String set) {
         this.set = set;
     }
+
+    //TODO
+    public String getBestSet() {
+        return set;
```

Comment:
Will be reworked in the future...need to determine the best way to go about managing set data